src/textual/widget.py

```diff
@@ -3733,12 +3733,12 @@ def _scroll_update(self, virtual_size: Size) -> None:
                 height - self.scrollbar_size_horizontal
             )
             if self.vertical_scrollbar._repaint_required:
-                self.call_later(self.vertical_scrollbar.refresh)
+                self.vertical_scrollbar.refresh()
```

Comment:
There should be a policy that every time `call_later` is used in Textual we must write a comment explaining why 😄 

I've certainly had a number of issues in the past where the scrollbars wouldn't update (size/position of them) until they were interacted with. Not sure if the `call_later` was resolving that.

Reply:
Yeah, probably.

The refresh defers the update anyway, so I don't see what the call_later was doing. And I couldn't get anything to break without it.